### ISSUE TITLE
Missing .svg from thumbnails and script to generate png files from variations of these thumbnails.

### ISSUE THREAD

**Issue by antrrax** [linuxmint/mint-themes#327]
Scripts do not generate thumbnails of Mint-Y variations that are displayed in the cinnamon interface, when creating a new color variation the thumb used is the default green.

mint-themes-master/src/Mint-Y/gtk-3.0/thumbnail.png
mint-themes-master/src/Mint-Y/cinnamon/mint-y-thumbnail.png
mint-themes-master/src/Mint-Y/gtk-3.0/thumbnail-dark.png
mint-themes-master/src/Mint-Y/cinnamon/mint-y-dark-thumbnail.png

They could do the same with this github project:
https://github.com/SebastJava/mint-yz-theme

in the 'mint-yz-theme-newflat/src/Mint-Y/gtk-3.0' folder in the assets.svg file he added the image of the thumbnails controls.
https://github.com/SebastJava/mint-yz-theme/blob/newflat/src/Mint-Y/gtk-3.0/assets.svg

and in the folder 'mint-yz-theme-newflat/src/Mint-Y/cinnamon' there is the file:
**mint-y-thumbnails-src.svg**
https://github.com/SebastJava/mint-yz-theme/blob/newflat/src/Mint-Y/cinnamon/mint-y-thumbnails-src.svg

and the script:
**render-cin-thumbnails.sh**
that generate the desktop thumbnail

------------
**Please add this option so when creating a new Mint-Y theme variation it will automatically create thumbnail files in the corresponding color.**

ex: if I put the script below inside the mint-themes-master folder and run it. A new 'blue-mod' variant will be created but with the thumbnails in green

```
#!/usr/bin/env bash
set -e

VARIATION_NAME="Blue-mod"
HUE_VALUE=210

COLOR_X="#3770aa"
COLOR_X_RGB="55, 112, 170"

COLOR_Y_1="#3770aa"
COLOR_Y_2="#35679a"
COLOR_Y_3="#4d7cab"
COLOR_Y_4="#1d4a77"

CURDIR=$PWD

sed -i "s/HUE\[\"Teal\"\] = 168/HUE\[\"Teal\"\] = 168\nHUE\[\"$VARIATION_NAME\"\] = $HUE_VALUE/g" "$CURDIR/colorize.py"

echo -e "\nx_hex_colors[\"$VARIATION_NAME\"] = \"$COLOR_X\"\nx_rgb_colors[\"$VARIATION_NAME\"] = \"$COLOR_X_RGB\"\n\ny_hex_colors1[\"$VARIATION_NAME\"] = \"$COLOR_Y_1\"\ny_hex_colors2[\"$VARIATION_NAME\"] = \"$COLOR_Y_2\"\ny_hex_colors3[\"$VARIATION_NAME\"] = \"$COLOR_Y_3\"\ny_hex_colors4[\"$VARIATION_NAME\"] = \"$COLOR_Y_4\"" >>  "$CURDIR/constants.py"

sed -i "s/or 'All'/, '$VARIATION_NAME' or 'All'/g" "$CURDIR/update-variations.py"
sed -i "s/, \"All\"/, \"$VARIATION_NAME\", \"All\"/g" "$CURDIR/update-variations.py"

"$CURDIR/update-variations.py" "$VARIATION_NAME"

"$CURDIR/generate-themes.py"

cd "$CURDIR/usr/share/themes/"
tar -zcf "$CURDIR/$VARIATION_NAME.tar.gz" "Mint-Y-$VARIATION_NAME" "Mint-Y-Dark-$VARIATION_NAME"

```
<img src="https://iili.io/5fpx5v.png" alt="5fpx5v.png" border="0">

**Comment by JosephMcc** [comment]
The tools to generate thumbnails are a part of Cinnamon. They are there for a reason. So they are available to create thumbnails for `cinnamon-settings themes`. There is no real reason in our case to duplicate those here.

**Comment by SebastJava** [comment]
@JosephMcc You clearly misunderstood this issue. I am 100% sure about this.

Okay, the title is a bit confusing. But the OP screenshot is very clear. @antrrax is talking about all those:

**/usr/share/themes/Mint-Y-COLOR/gtk-3.0/thumbnail.png
/usr/share/themes/Mint-Y-COLOR/cinnamon/thumbnail.png**
That's 2 different thumbnails for every color variant = 22 thumbnails.

And those are currently being created by hand, one-by-one, for every time someone wants to modify or add a new color variant. There are no tools to generate these! I have these tools. I have been using them for years now. There is a group of pull requests still waiting for this: PR number 375-379. https://github.com/linuxmint/mint-themes/pulls

Yes, that's a group. At first, long ago, there was only one big PR, but I thought you might find this too big. So, I divided it into a group of small PRs. But then, as years were passing by, it got outdated. So, I created this third group of PRs. That was probably my final attempt.

And there is still this https://github.com/linuxmint/mint-themes/pull/364 waiting. All those PRs would really help in creating new color variants.

But of course, you consider there is absolutely no need to create new color variants, as the current ones are considered to be "top-most professional colors". But many people, including me, strongly, totally disagree with this. And this is not only a subjective thing. I have made some precise measurements, clearly demonstrating these colors should be changed. All of them, IMHO. It is all explained, in details, here: https://github.com/SebastJava/mint-yz-theme#readme

I did one big mistake, years ago. I did not have enough experience, and self-confidence, and equipment to show you strong arguments for a big change on all colors. But now it's different. I would have many things to say but I am pretty sure you would not take the time to read it anyway.
